crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -18,15 +18,17 @@
 
 module Api
   class Crowbar < Tableless
-    attr_reader :version
+    attr_reader :version, :addons
```

Comment:
this is no longer relevant, as it got dropped in the switch to class methods (should have been squashed somehow)